test/lib/prisma.test.ts

```diff
@@ -0,0 +1,101 @@
+
+import {it, expect} from '@jest/globals';
+import {whereAndSelect} from "@lib/prisma";
+
+it('can decorate using whereAndSelect', async () => {
+  whereAndSelect( (queryObj) => {
+    expect(queryObj).toStrictEqual({ where: { id: 1 }, select: { example: true } });
+    },
+    { id: 1 },
+    [
+      "example"
+    ])
```

Comment:
Codacy has a fix for the issue: [Replace `··])` with `);`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7713215)
```suggestion
  );
```